.github/workflows/build-linux-arm64-installer.yml

```
@@ -101,12 +102,22 @@ jobs:
           return release.data.tag_name;
 
     - name: Get latest bladebit plotter
+      if: '!github.event.release.prerelease'
       run: |
         mkdir "$GITHUB_WORKSPACE/bladebit"
         wget -O /tmp/bladebit.tar.gz https://github.com/Chia-Network/bladebit/releases/download/${{ steps.latest-bladebit.outputs.result }}/bladebit-${{ steps.latest-bladebit.outputs.result }}-ubuntu-arm64.tar.gz
         tar -xvzf /tmp/bladebit.tar.gz -C $GITHUB_WORKSPACE/bladebit
         chmod +x "$GITHUB_WORKSPACE/bladebit/bladebit"
 
+    - name: Get latest prerelease bladebit plotter
+      if: github.event.release.published && '!github.event.release.prerelease'
```

```suggestion
      if: github.event.release.published && 'github.event.release.prerelease'
```

I think for prereleases we'd want the prerelease bladebit (I think this logic might be backwards?)